host/xtest/xtest_1000.c

```diff
@@ -1182,3 +1182,11 @@ static void xtest_tee_test_1013(ADBG_Case_t *c)
 
 	TEEC_ReleaseSharedMemory(&shm);
 }
+
+static void xtest_tee_test_1013(ADBG_Case_t *c)
+{
+	int i;
+	for (i = 0; i < 50; i++)
+		xtest_tee_test_1013_single(c);
+}
+
```

Comment:
The problem I see with this: it will cause 50 times more text output for the test, and it will count 50 times more sub-cases. IMO it would be better to loop inside `xtest_tee_test_1013()` -- you could extract the two scenarios (busy loop/hashing) and iterate over them.